Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/ViewModel/MapViewModel.swift

```diff
@@ -20,6 +20,9 @@ class MapViewModel: ObservableObject {
         }
     }
     private var updatedLocations: Set<CLLocation> = []
+    var useMapBox: Bool {
+        SettingsModel.shared.useMapBox
```

Comment:
Similar point to the above - maybe we should call vars like that `useMapBoxMap` instead?

Reply:
changed